cmd/infra/powervs/create.go

```diff
@@ -1553,7 +1558,7 @@ func (infra *Infra) createTransitGateway(ctx context.Context, options *CreateInf
 	tg, _, err = tgapisv1.CreateTransitGatewayWithContext(ctx, &transitgatewayapisv1.CreateTransitGatewayOptions{
 		Location:      utilpointer.String(options.TransitGatewayLocation),
 		Name:          utilpointer.String(transitGatewayName),
-		Global:        utilpointer.Bool(true),
+		Global:        utilpointer.Bool((!(options.Region == options.VPCRegion)) || options.GlobalRoutingEnabled),
```

Comment:
it won't be equal, the notation that powervs follows is different than the one followed by vpc.
Need to use the region mapping from https://github.com/ppc64le-cloud/powervs-utils.
Lets wait for this https://github.com/ppc64le-cloud/powervs-utils/pull/11 PR to get merged, post that, we can use it for checking whether they belong to the same region.